plugins/SitesManager/API.php

```diff
@@ -1346,6 +1358,68 @@ public function setDefaultTimezone($defaultTimezone)
         return true;
     }
 
+    /**
+     * Sets global query parameter exclusion based on the specified exclusion type.
+     *
+     * @param string $exclusionType The type of query param exclusion, must be of the following:
+     *  - no_exclusions
+     *  - common_pii_exclusions
+     *  - custom_exclusions
+     * @param string|null $queryParamsToExclude (Optional) List of query parameters to exclude when $exclusionType is 'custom'.
```

Comment:
```suggestion
     * @param string|null $queryParamsToExclude (Optional) Comma separated list of query parameters to exclude when $exclusionType is 'custom'.
```

Can't be bad to clarify that "list" is specifically a comma separated list, and should not be confused with an array or anything else.